src/commands/punsubscribe.json

```
@@ -1,7 +1,7 @@
 {
     "PUNSUBSCRIBE": {
         "summary": "Stops listening to messages published to channels that match one or more patterns.",
-        "complexity": "O(N) where N is the number of patterns the client is already subscribed.",
+        "complexity": "O(N) where N is the number of patterns to unsubscribe to.",
```

```suggestion
        "complexity": "O(N) where N is the number of patterns to unsubscribe.",
```
we can maybe add `from`, but not `to`.